gensim/test/test_corpora.py

```diff
@@ -16,14 +16,17 @@
 
 import numpy
 
-from gensim.utils import to_unicode, smart_extension
+from gensim.utils import to_unicode  # , smart_extension
```

Comment:
Done.

Side Note: There are many more unused imports throughout gensim. They can be dangerous to remove, though, for someone like me unfamiliar with the internals of those packages being imported. For example `import seaborn` has side-effects, and obviously `from future import division` does too.